libs/shared/design-tokens/BUILD.bazel

```
@@ -12,3 +14,24 @@ ts_library(
         "@npm//tslib",
     ],
 )
+
+filegroup(
+    name = "design-tokens-entrypoints",
+    srcs = glob(["src/lib/global/*.yml", "src/lib/patterns/*.yml"])
+)
+
+filegroup(
+    name = "design-tokens-aliases-entrypoints",
+    srcs = glob(["src/lib/aliases/*.yml"])
+)
+
+generate_color_palette(
+    name = "color_palette",
+    palette_aliases = "src/lib/aliases/palette-source.alias.yml",
+)
+
+build_design_tokens(
+  name = "build",
+  entrypoints = [":design-tokens-entrypoints"],
+  aliases_entrypoints = [":design-tokens-aliases-entrypoints", ":color_palette"]
+)
```

You don't need the filegruops you can pass the glob directly in:


```suggestion
build_design_tokens(
  name = "design-tokens",
  entrypoints = glob(["src/lib/aliases/*.yml"]),
  aliases_entrypoints = glob(["src/lib/global/*.yml", "src/lib/patterns/*.yml"]) + [":src/lib/aliases/palette-source.alias.yml"]
)
```

I think it would be more readable